utils/errors/notifier/notifier.go

```diff
@@ -109,21 +122,26 @@ func convToSentry(in errors.ErrorExt) *raven.Stacktrace {
 	return out
 }
 
-func parseRawData(ctx context.Context, rawData ...interface{}) map[string]interface{} {
-	m := make(map[string]interface{})
+func parseRawData(ctx context.Context, rawData ...interface{}) (extraData map[string]interface{}, tagData []map[string]string) {
```

Comment:
Does `tagData` need to be a slice of maps?

Reply:
for cases when the error wraps another error and each of them has tags respectively.

Reply:
The reason not to unfold and fold all tags into one map takes a for loop is when later `packet.AddTags` was called, it will unfold all tags again.